gluster/block/auth/openshift-clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: glusterblock-provisioner-runner
```

Comment:
just curious, have you tested this works?

I know some 'secret' permissions will come later when iscsi chap is enabled